src/Card/Card.stories.js

```diff
@@ -318,6 +301,45 @@ const CardTemplateMinimalRight = ({ ...rest }) => (
     </Card.Content>
   </Card>
 );
+
+const CardTemplateCustom = ({ ...rest }) => (
```

Comment:
```suggestion
const CardTemplateCustom = (props) => (
```

Reply:
This one is just for the example template